src/app/routes/getInitialData.test.jsx

```diff
@@ -32,11 +32,15 @@ describe('getInitialData', () => {
   routes
     .filter(route => !pageTypesToSkip.includes(route.pageType))
     .forEach(({ getInitialData, pageType }) => {
+      const path =
+        pageType === 'liveRadio'
+          ? '/korean/bbc_korean_radio/liveradio'
+          : MOCK_PATH;
```

Comment:
Out of interest, why is the required?

Reply:
This is because the ID it passes to the BFF is extracted, without changing the path to the expected format, the test would fail https://github.com/bbc/simorgh/blob/4ed713ff094b9ef6b7ddbae8b67eb15d86dc2018/src/app/routes/utils/constructPageFetchUrl/index.ts#L101-L107